content/en/getting_started/cloud_siem/_index.md

```diff
@@ -50,7 +50,7 @@ This guide walks you through best practices for getting started with Cloud SIEM.
 
 ## Phase 1: Setup
 
-1. Configure [log ingestion][2] to collect logs from your sources. Review [Best Practices for Log Management][3].
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** `‏`[2] seems to be not used anymore. We could remove it?

Reply:
We're using a different link now, so I've replaced it with the new link.